fury/actor.py

```diff
@@ -2452,3 +2452,73 @@ def texture_on_sphere(rgb, theta=60, phi=60, interpolate=True):
     earthActor.SetTexture(atext)
 
     return earthActor
+
+
+def sdf(centers, directions=(1, 0, 0), colors=(255, 0, 0), primitive='torus'):
```

Comment:
Can you please elaborate?

Reply:
Sure thing! What I'm suggesting is to have a `def torus_sdf(args)` actor or simply `def torus(args, render='sdf')` instead of a single function to render all the primitives.

Reply:
I recommend it the opposite so I think, we should talk about it during the next meeting. Please, keep it as it is for the moment @lenixlobo